db/migrations/postgres/000109_fix_tokentransfer_index.up.sql

```diff
@@ -2,6 +2,6 @@ BEGIN;
 DROP INDEX tokentransfer_protocolid;
 DROP INDEX tokenapproval_protocolid;
 
-CREATE UNIQUE INDEX tokentransfer_protocolid ON tokentransfer(namespace, connector, protocol_id);
-CREATE UNIQUE INDEX tokenapproval_protocolid ON tokenapproval(namespace, connector, protocol_id);
+CREATE UNIQUE INDEX tokentransfer_protocolid ON tokentransfer(namespace, pool_id, protocol_id);
+CREATE UNIQUE INDEX tokenapproval_protocolid ON tokenapproval(namespace, pool_id, protocol_id);
```

Comment:
Well I ended up with two other brand new database migrations anyway, so I might as well roll these lines into 110 or 111 instead of modifying this one.